### PR TITLE
issue: 1338066 Fix disable flow tag for unicast flows

### PR DESCRIPTION
**Pull request by serglelv**
Disable the flow tag if setsockopt(SO_REUSEADDR) has been set

### CODE REVIEW

**Comment by serglelv** [comment]
@olgk  Please review

**Comment by serglelv** [comment]
@igor-ivanov Please review and add it to the vNext

**Comment by liranoz12** [comment]
@serglelv Please approve

File: src/vma/dev/ring_simple.cpp [hunk 1]
```diff
@@ -402,6 +402,10 @@ bool ring_simple::attach_flow(flow_tuple& flow_spec_5t, pkt_rcvr_sink *sink)
 	/* Get the appropriate hash map (tcp, uc or mc) from the 5t details */
 	if (flow_spec_5t.is_udp_uc()) {
 		flow_spec_udp_key_t key_udp_uc(flow_spec_5t.get_dst_ip(), flow_spec_5t.get_dst_port());
+		if(flow_tag_id && si->flow_in_reuse()) {
```
Comment (igor-ivanov) [inline src/vma/dev/ring_simple.cpp]: coding style /if(/if (/

**Pull request merged**